jenkins/controller/plugins.txt

```diff
@@ -17,11 +17,15 @@
 # be up to date with what is in the master branch. You'll have to
 # look at the release branch for the release of OpenShift you are on.
 # [1] https://github.com/openshift/jenkins/blob/master/2/contrib/openshift/base-plugins.txt
-basic-branch-build-strategies
-generic-webhook-trigger
-github-oauth
-kubernetes-credentials-provider
-pipeline-github
-pipeline-utility-steps
-slack
-timestamper
+basic-branch-build-strategies:1.3.2
+generic-webhook-trigger:1.84.2
+github-oauth:0.39
+kubernetes-credentials-provider:1.199.v4a_1d1f5d074f
+pipeline-github:2.8-138.d766e30bb08b
+pipeline-utility-steps:2.13.0
+slack:625.va_eeb_b_168ffb_0
+timestamper:1.20
+
+# The below list are plugins that are also in base-plugins.txt but for
+# some reason or other we need to temporarily freeze or fast-track.
+token-macro:293.v283932a_0a_b_49
```

Comment:
Can we add a comment for this with some of what's in the commit message?

Reply:
updated